yoti-sdk-api/src/main/java/com/yoti/api/client/ActivityDetails.java

```diff
@@ -39,6 +39,14 @@
      */
     String getRememberMeId();
 
+    /**
+     * Return the parent-remember-me ID, which is a unique, stable identifier for a user in the context of an organisation/user
+     * interaction. You can be use it to identify returning users.  This value is consistent for a given user across different apps
+     * belonging to a single organisation.
+     *
+     * @return user ID.
```

Comment:
```suggestion
     * @return Parent Remember Me ID.
```

Looks like the other `@return`s in this file need updating too

Reply:
Well spotted, fixed them